### PR TITLE
Fixissue9

### PR DESCRIPTION
**Pull request by GitHubUser5136**

### CODE REVIEW

File: src/sample/stuDashController.java [hunk 1]
```diff
@@ -454,10 +446,10 @@ public void validateCGPA(KeyEvent e) {
...
       }
     } else {
-      Messages.setText("CGPA cannot be empty!");
-      saveButton.setDisable(true);
-      name.setDisable(true);
-      nric.setDisable(true);
+      //Messages.setText("CGPA cannot be empty!");
+      saveButton.setDisable(false);
+      name.setDisable(false);
+      nric.setDisable(false);
```
Comment (Michelle-Lohwt) [inline src/sample/stuDashController.java:452]: why false?

File: src/sample/stuDashController.java [hunk 2]
```diff
@@ -454,10 +446,10 @@ public void validateCGPA(KeyEvent e) {
         }
       }
     } else {
-      Messages.setText("CGPA cannot be empty!");
-      saveButton.setDisable(true);
-      name.setDisable(true);
-      nric.setDisable(true);
+      //Messages.setText("CGPA cannot be empty!");
```
Comment (Michelle-Lohwt) [inline src/sample/stuDashController.java]: remove comments ah

File: src/sample/lecDashController.java [hunk 3]
```diff
@@ -233,7 +234,7 @@ public void validateNRIC(){
       } else{
         Message.setText("");
         saveButton.setDisable(false);
-        name.setDisable(true);
+        name.setDisable(false);
```
Comment (Michelle-Lohwt) [inline src/sample/lecDashController.java:225]: why false?

**Review by Michelle-Lohwt — CHANGES_REQUESTED** [review]
Check the comments

File: src/sample/lecDashController.java (hunk 3 above)
Comment (GitHubUser5136) [reply src/sample/lecDashController.java:225]: Because the nric got no error le, so they can edit their name

Comment (GitHubUser5136) [reply src/sample/lecDashController.java:225]: If nric got error, they cannot edit name

File: src/sample/stuDashController.java (hunk 1 above)
Comment (GitHubUser5136) [reply src/sample/stuDashController.java:452]: because the cgpa format is correct le, so they can save their info or edit their name and nric

**Pull request merged**